internal/services/manageddevopspools/managed_devops_pool_data_source.go

```diff
@@ -0,0 +1,381 @@
+			Type:     pluginsdk.TypeList,
+			Computed: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"manual_resource_predictions_profile": manualResourcePredictionsProfileSchemaComputed(),
+
+					"automatic_resource_predictions_profile": automaticResourcePredictionsProfileSchemaComputed(),
+				},
+			},
+		},
+
+		"tags": commonschema.TagsDataSource(),
+
+		"vmss_fabric_profile": {
+			Type:     pluginsdk.TypeList,
+			Computed: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"image": {
+						Type:     pluginsdk.TypeList,
+						Computed: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"aliases": {
+									Type:     pluginsdk.TypeList,
+									Computed: true,
+									Elem: &pluginsdk.Schema{
+										Type: pluginsdk.TypeString,
+									},
+								},
+
+								"buffer": {
+									Type:     pluginsdk.TypeString,
+									Computed: true,
+								},
+
+								"resource_id": {
+									Type:     pluginsdk.TypeString,
+									Computed: true,
+								},
+
+								"well_known_image_name": {
+									Type:     pluginsdk.TypeString,
+									Computed: true,
+								},
+							},
+						},
+					},
+
+					"os_profile": {
+						Type:     pluginsdk.TypeList,
+						Computed: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"logon_type": {
+									Type:     pluginsdk.TypeString,
+									Computed: true,
+								},
+
+								"secrets_management": {
+									Type:     pluginsdk.TypeList,
+									Computed: true,
+									Elem: &pluginsdk.Resource{
+										Schema: map[string]*pluginsdk.Schema{
+											"certificate_store_location": {
+												Type:     pluginsdk.TypeString,
+												Computed: true,
+											},
+
+											"certificate_store_name": {
+												Type:     pluginsdk.TypeString,
+												Computed: true,
+											},
+
+											"key_export_enabled": {
+												Type:     pluginsdk.TypeBool,
+												Computed: true,
+											},
+
+											"observed_certificates": {
+												Type:     pluginsdk.TypeList,
+												Computed: true,
+												Elem: &pluginsdk.Schema{
+													Type: pluginsdk.TypeString,
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+
+					"sku_name": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+
+					"storage_profile": {
+						Type:     pluginsdk.TypeList,
+						Computed: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"data_disk": {
+									Type:     pluginsdk.TypeList,
+									Computed: true,
+									Elem: &pluginsdk.Resource{
+										Schema: map[string]*pluginsdk.Schema{
+											"caching": {
+												Type:     pluginsdk.TypeString,
+												Computed: true,
+											},
+
+											"disk_size_gb": {
```

Comment:
Could you update this to `disk_size_in_gb` to follow our naming conventions?

```suggestion
											"disk_size_in_gb": {
```